s2i/assemble

```diff
@@ -54,19 +54,22 @@ if [ ! -z "$YARN_ENABLED" ]; then
 else
 	echo "---> Installing dependencies"
 	echo "---> Using 'npm install'"
-	npm install -d
+	npm install -s
 	npm run build
 	echo "---> Cleaning up npm cache"
 	rm -rf .npm
-	# echo "---> Setting directory write permissions"
-	# exec chmod -R g+rw .
 fi
 
-echo "---> Building and Installing Web Application from source..."
-if [ -f /tmp/src/nginx.conf ]; then
-  mv /tmp/src/nginx.conf /etc/nginx/nginx.conf
-fi
+DEPLOY_DIR=/opt/app-root/output
```

Comment:
Maybe I'm misremembering, but I thought this was going to be configurable?

Reply:
Yea, this is a little confusing,  the OUTPUT_DIR env var is the thing that is configurable.  since angular puts its compiled code in a different location than react, and ember and etc....

This thing, DEPLOY_DIR, is just a common place to copy the output from the OUTPUT_DIR, so i don't have to do something like this in the changed build,  https://gist.github.com/lholmquist/91729dd36ea73747b22d074769d81af1#file-angular-chain-build-yaml-L122

It is techinally not needed, but it would make it easier. I think,


I'm also open to suggestions to where it should be saved.